common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -553,6 +553,29 @@ public UTF8String trim() {
     return copyUTF8String(s, e);
   }
 
+  /**
+   * Trims whitespaces (<= ASCII 32) from both ends of this string.
```

Comment:
Interesting, so SQL trim just removes space, and java.lang.String.trim() removes everything <= 32. Maybe we could refer to that in this doc, that this is the purpose of this additional trim method.

Reply:
yea, let's mention it's the same as ` java.lang.String.trim`